cluster/bin/snappy-job.sh

```diff
@@ -127,7 +127,7 @@ while (( "$#" )); do
         echo "The netrc file $passwordfile not found."
         exit 1
       fi
-      securePart=" --netrc-file ${passwordfile}"
+      securePart=" --config ${passwordfile}"
```

Comment:
Yes, config file can take any config and not just the credentials.
But the --netrc-file does not seem to be working with snappy-job.sh. Couldn't figure out what the problem was, as it was not working even with explicit curl command. So switched to this option.

Is there any concern with --config?